release/ray_release/config.py

```diff
@@ -50,9 +51,11 @@ class TestDefinition(dict):
 
 RELEASE_PACKAGE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
 
-RELEASE_TEST_SCHEMA_FILE = os.path.join(
-    RELEASE_PACKAGE_DIR, "ray_release", "schema.json"
-)
+RELEASE_TEST_SCHEMA_FILE = bazel_runfile("release/ray_release/schema.json")
```

Comment:
[1] RELEASE_TEST_SCHEMA_FILE = bazel_runfile(
           "release/ray_release/schema.json", 
           repo_dir = os.path.join(os.path.dirname(__file__), "..", "..")
     )

Reply:
ok, I tried this, the part I do not like about this change is that I do not even want to execute `os.path.join()` or the path searching logic when runfiles exist.

that said, I got another way to fallback, which is falling back this repo's root.